app/content/models/notification.py

```diff
@@ -1,11 +1,10 @@
 from django.db import models
 
-from app.content.models.user import User
 from app.util.models import BaseModel
 
 
 class Notification(BaseModel):
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    user = models.ForeignKey("content.User", on_delete=models.CASCADE)
```

Comment:
Vi burde begynne å bruke `settings.AUTH_USER_MODEL` i stedet for å hardkode med app-navnet. Det gjør det lettere å endre på ting hvis vi skulle trenge det